client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -161,6 +221,17 @@ class InlineHelpSearchResults extends Component {
 		);
 	}
 
+	getNewRailcarSeed() {
```

Comment:
`getNewRailcarSeed` seems to be generally applicable.  Can / should this go in a utility lib?

Reply:
Good call, especially since it's also used in domains (where I got it from 😬). Moved it into a utils module in 2f01b2f.

Reply:
I don't understand what the seed is being used for.